src/Popover/helpers/getScrollableParent.js.flow

```diff
@@ -0,0 +1,5 @@
+// @flow
+
+export type GetScrollableParent = (node: React.node) => React.node;
```

Comment:
`HTMLElement` should be probably used. `React.Node` is completely different type.